tasks/src/main/scala/dagr/tasks/misc/MoveFile.scala

```diff
@@ -0,0 +1,10 @@
+package dagr.tasks.misc
+
+import java.nio.file.Path
+
+import dagr.core.tasksystem.ShellCommand
+
+/**
+  * Uses the shell to move a file or directory from one path to another
+  */
+class MoveFile(val from: Path, val to: Path) extends ShellCommand("mv", from.toString, to.toString)
```

Comment:
I know ... I debated just using ShellCommand directly, but I feel like for common file manipulation stuff it's useful to have dedicated tasks that we can improve over time.